### ISSUE TITLE
svglinkify.py does not work in Ubuntu 18.04 / libpoppler 0.62

### ISSUE THREAD

**Issue by rartino** [oxplot/gists#3]
Using svglinkify.py with the version of Inkscape (0.92.3-1) + dependencies (likely most importantly libpoppler 0.62) distributed in Ubuntu 18.04 results in a pdf with no clickable links.

svglinkify.py runs without reporting errors. If I add a print statement showing the rectangles, it seem to find them ok. Still, the generated output just misses the clickable links. However, if I keep everything the same, still running svglinkify.py on Ubuntu 18.04, *but*, I use a pdf file generated from Inkspace + dependencies running on Ubuntu 17.10, the links show up as desired.

Downgrading Inkscape itself to the exact version from Ubuntu 17.10 (0.92.2-1build1) did not resolve the issue. Hence, I suspect it is a change that has happened between libpoppler68 (v 0.57) and libpoppler73 (v 0.62). 

A workaround is to generate the pdf used for svglinkify using the inkscape version in the snap store instead, i.e., install it with;
```
  sudo snap install inkscape
```
and run it with:
```
  snap run inkscape
```
This version uses libpoppler58 (v 0.41) and everything seems to be working with this.

**Comment by am97** [comment]
I have a similar issue on Fedora 28. svglinkify.py works, but the clickable areas coordinates are flipped vertically.
Here is an example: [example.zip](https://github.com/oxplot/gists/files/2522091/example.zip)
The command used is `/path/to/svglinkify.py drawing.svg drawing.pdf links.pdf`
Fedora 28 ships with libpoppler v 0.62.

**Comment by oxplot** [comment]
@rartino @am97 thanks for reporting these. I'll look at that example and see if it can be worked around.

**Comment by oxplot** [comment]
I had a quick look and I'm going to abondon the current `svglinkify` which is very brittle and requires a lot of manual intervention, and instead focus on doing https://github.com/oxplot/gists/issues/4 .

**Comment by oxplot** [comment]
[Pre-release of of fairly automatic svglinkify is up](https://github.com/oxplot/svglinkify/releases/tag/v0.1).

Please give it shot and let me know how it goes. The new program does not not require any kind of magenta box hackery.

**Comment by am97** [comment]
It works! And it's much simpler now, thanks!

**Comment by rartino** [comment]
This worked for me as well, thanks!

**Comment by oxplot** [comment]
Great

**Comment by rartino** [comment]
Not to, in any way, take away from the very nice work with svglinkify that saved a project I worked on.

But, as a small note: it seems we could get native support for pdf link export in inkscape relatively soon. It seems pdf link support was added to the cairo library (which inkspace uses) in v1.16:
  https://bugs.freedesktop.org/show_bug.cgi?id=40078
and v1.16 was released Oct 19, 2018. So, maybe at least in 2019 we'll see distributions shipping with this.
